src/platform_layers/simulator_platform_layer/src/simulator_adu_core_exports.cpp

```diff
@@ -1,189 +0,0 @@
-/**
```

Comment:
one thing I liked about the simulator example (although im biased) is that it provided a simple template to create a new "handler".  Do we have something similar currently?

Reply:
No. My plan is to componentize this further to make it easier for a Device Bulder to implement each component separately. When that happen, we have a better docs and samples for each extensions.